download-latest.sh

```diff
@@ -65,6 +65,30 @@ get_os() {
     return 0
 }
 
+# Gets the C lib the system is using by setting the $c_lib variable.
+# Returns 0 in case of success, 1 otherwise.
+get_C_library(){
+  if [ "$os" != 'linux' ] ; then
+    return 0
+  fi
+  if ldd --version | grep -qi eglibc; then
+    c_lib='eglibc'
+    return 0
+  elif ldd --version | grep -qi glibc; then
+    c_lib='glibc'
+    return 0
+  elif ldd --version | grep -qi musl; then
+    c_library_failure_usage
```

Comment:
you already display the failure usage in the function in `main`, so don't display it here.